ouroboros-consensus/src/Ouroboros/Consensus/BlockFetchServer.hs

```diff
@@ -106,6 +108,9 @@ blockFetchServer _tracer chainDB registry = senderSide
 
 -- | Events traced by the Block Fetch Server.
 data TraceBlockFetchServerEvent blk
-   -- TODO no events yet. Tracing the messages send/received over the network
-   -- might be all we need?
+  = TraceNoBlocks (Point blk) (Point blk)
```

Comment:
While you're at it, it would be useful to include a message for starting a batch (and completing one?)

Reply:
Done